README.md

```diff
@@ -10,6 +10,7 @@ Swarm control planes.
 ## AWS
 - v12
   - v12.2
+    - [v12.3.0](https://github.com/giantswarm/releases/tree/master/aws/v12.3.0)
```

Comment:
This should be under v12->v12.3, not under v12->v12.2 .

Reply:
Good catch, I fixed it now.